scripts/index-services.py

```diff
@@ -2,11 +2,11 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
+    index-services.py <stage> --frameworks=<frameworks> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
 
-    --serial        Do not run in parallel (useful for debugging)
-    --index=<index>  Search API index name [default: g-cloud]
-    --frameworks=<frameworks> Optional comma-separated list of framework slugs that should be indexed
+    --serial                  Do not run in parallel (useful for debugging)
+    --index=<index>           Search API index name [default: g-cloud]
+    --frameworks=<frameworks> Comma-separated list of framework slugs that should be indexed
```

Comment:
I *think* that this is a list of the possible `[options]`, so if frameworks is required it should probably be removed from this list of possible options. (But I might be wrong... I don't really know the ins and outs of docopt).

Would be nice to keep in the comment somewhere that `<frameworks>` is a comma-separated list though.